Libplanet/Net/Swarm.cs

```diff
@@ -404,11 +429,26 @@ private set
             try
             {
                 tasks.Add(BroadcastTxAsync(broadcastTxInterval, _cancellationToken));
-                tasks.Add(Task.Run(() => _poller.Run(), _cancellationToken));
                 tasks.Add(
                     _protocol.RefreshTableAsync(TimeSpan.FromSeconds(10), _cancellationToken));
                 tasks.Add(
                     _protocol.RebuildConnectionAsync(TimeSpan.FromMinutes(30), _cancellationToken));
+                tasks.Add(
+                    Task.Run(() =>
+                    {
+                        // Ignore NetMQ related exceptions during NetMQPoller.Run() to stabilize
+                        // tests.
```

Comment:
In real situation not test, this should still be ignored when `_poller.Run()` fails?

Reply:
I guess it's okay because we ignore only `TerminatingException` and `ObjectDisposedException` that be raised from NetMQ during disposing of itself.